docs/bdd/index.rst

```diff
@@ -0,0 +1,24 @@
+The BDD Guide
+=============
+
+Behaviour driven development is an approach to software development process that provides software development and management teams
+with shared tools and a shared process to collaborate on software development. The awesome part of BDD is its ubiquitous language,
+which is used to describe the software in English-like sentences of domain specific language.
```

Comment:
Not necessary `English-like`. `Bussiness understandable`?

Reply:
"domain specific language" == "understandable for business side"